frontend/Views/Config.tsx

```diff
@@ -15,22 +14,25 @@ for (let i = 0; i < 11; i++) {
 	let newGame: Game = new Game('Super Metroid');
 	mockData.push(newGame);
 }
-
-const gameList = function(gameData: Game[]): JSX.Element[] {
-	let result = [];
-	for (let game of gameData) {
-		result.push(<>
-			<GamesCheck game={game.name}/>
-		</>);
+interface GameListProps{
+	games: Game[];
+}
+class GameList extends React.Component<GameListProps> {
```

Comment:
You may want to change this to a functional component, since it has no state.

```
const GameList : React.FC<GameListProps> = (props) => {
```

Reply:
btw, generally always go for functional components rather than class-based components, unless you need state - in which case you have the option of turning your functinal component into a class, or trying hooks (but let's skip hooks for now).